tests/functional/test_install.py

```diff
@@ -657,12 +657,10 @@ def test_install_package_with_prefix(script, data):
         '--no-binary', 'simple', '--no-index', 'simple==1.0',
     )
 
-    if hasattr(sys, "pypy_version_info"):
-        path = script.scratch / 'prefix'
-    else:
-        path = script.scratch / 'prefix' / 'lib' / 'python{0}'.format(pyversion)  # noqa
+    rel_prefix_path = script.scratch / 'prefix'
     install_path = (
-        path / 'site-packages' / 'simple-1.0-py{0}.egg-info'.format(pyversion)
+        distutils.sysconfig.get_python_lib(prefix=rel_prefix_path) /
```

Comment:
Not sure how good an idea it is to be depending on/using distutils here and in a few other places; is there some equivalent function in pip's codebase?

Reply:
We already use it in other places, and in pip's codebase. I don't see an equivalent function we could use.

Reply:
Thanks for looking! I now think my question was not exactly specific to this PR anyway.